static/js/util/courseList.js

```diff
@@ -25,7 +25,18 @@ function asPercent(number) {
 /**
  * Determine React elements for the UI given a course status
  */
-export function makeCourseStatusDisplay(course: Object, now: moment = moment()): string|React$Element {
+export type Course = {
+  runs: Array<CourseRun>;
+  status: string;
+};
+export type CourseRun = {
```

Comment:
I think most of these should be optional. Can you enable `// @flow` on `courseList_test.js` and fix the warnings for that file?